app/views/user/details.phtml

```diff
@@ -1,9 +1,11 @@
 <?php $this->partial('aside_configure'); ?>
 
+<?php $isAdmin = $this->details['is_admin']; ?>
+
 <div class="post">
     <a href="<?= _url('user', 'manage'); ?>"><?= _t('admin.user.back_to_manage'); ?></a>
 
-    <legend><?= $this->username; ?></legend>
+    <legend><?= $this->username ?><?php if ($isAdmin) echo ' &#x2015; ', _t('admin.user.admin'); ?></legend>
```

Comment:
```suggestion
    <legend><?= $this->username ?><?php if ($isAdmin) echo ' ― ', _t('admin.user.admin'); ?></legend>
```

I prefer Unicode than entities when possible :-)

Reply:
I'll change that